src/main/java/homeworks/puppyes.java

```diff
@@ -0,0 +1,41 @@
+package homeworks;
+
+import java.util.Scanner;
+
+public class puppyes {
+    public static void main(String[] args) {
+        Scanner age = new Scanner(System.in);
+        System.out.print("Сколько лет вашей собаке сейчас?: ");
+        int a = age.nextInt();
+
+        Scanner mon = new Scanner(System.in);
+        System.out.print("Сколько месяцев вашей собаке сейчас?: ");
+        int m = mon.nextInt();
+
+        Scanner year = new Scanner(System.in);
+        System.out.print("Сколько лет вы хотите прибавить?: ");
+        int y = year.nextInt();
+
+        Scanner month = new Scanner(System.in);
```

Comment:
object scanner can be created only one time and then you can use it again and again) so no needed to create it 3 times